### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by alfredoC10**
Hello, talented code reviewers! it's such a honor to finally have your sharp observations in our code! This time we worked in the Responsive design project that was a great experience to understand more about Media queries. The features used:
-HTML & CSS,
- Background images
- Display grid
- Display flex
- Media Queries
- percentages 
- CSS Viewport units 
Excellent friday to you all! I hope work has been gratifying!

### CODE REVIEW

File: CSS/media.css [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    position: relative;
+  }
+
+  .big-logo-date {
+    display: none;
+  }
+
+  .big-title {
+    /* position: absolute;
+    bottom: 47px; */
```
Comment (SpaYco) [inline CSS/media.css]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/

File: CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,904 @@
...
+}
+
+.colorAAA {
+  color: #aaa;
+}
+
+/* .flexbox-1 {
+  display: flex;
+  flex-direction: column;
+} */
```
Comment (SpaYco) [inline CSS/style.css]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/

File: index.html [hunk 3]
```diff
@@ -0,0 +1,702 @@
...
+  <link rel="stylesheet" href="./CSS/media.css" type="text/css">
+  <link href="https://fonts.googleapis.com/css2?family=Nova+Square&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Rajdhani:wght@500&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/13a4f3443f.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <div class="chair">
+    <i class="fas fa-wheelchair"></i>
+  </div>
+  <!-- <div class="flexbox-1"> -->
```
Comment (SpaYco) [inline index.html]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] The Top picture will display behind the navbar on mobile screens

image




## Optional 


* [ ] Use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links
* [ ] Use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: CSS/media.css [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+
+  .grid-1 {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(3, 1fr);
+    column-gap: 8px;
+    row-gap: 8px;
+    height: auto;
+    margin: 110px 7px 50px;
+  }
```
Comment (moinkhanif) [inline CSS/media.css:198]: TNWfan 33- Google Chrome 2020-09-13 01-36-32_Trim_1
**[OPTIONAL]** On smaller screens the first article within grid-1 seems to have issues. Kindly fix it.

Hint: Play around with max-height on `.grid-1` to check which works best.

**Review by moinkhanif — APPROVED** [review]
##  Status: Approved! 🥳  

Hey @Huemac-Alfredo, great work completing both mandatory and optional requests from the previous review! 🎉 Your project has been approved. You have a keen eye for details and I admire that.

[OPTIONAL]: I found that the first article behaves differently on smaller screens. I have mentioned it below in the comments. You could fix it in your free time. 👍 

You may now merge the branch.

Happy coding! 🐱‍🏍

**Pull request merged**